product_docs/docs/epas/16/application_programming/epas_compat_spl/12_working_with_collections/04_using_the_bulk_collect_clause/02_fetch_bulk_collect.mdx

```diff
@@ -6,56 +6,59 @@ redirects:
 
 <div id="fetch_bulk_collect" class="registered_link"></div>
 
-You can use the `BULK COLLECT` clause with a `FETCH` statement. Instead of returning a single row at a time from the result set, the `FETCH BULK COLLECT` returns all rows at once from the result set into the specified collection unless restricted by the `LIMIT` clause:
+You can use the `BULK COLLECT` clause with a `FETCH` statement. Instead of returning a single row at a time from the result set, the `FETCH BULK COLLECT` returns all rows at once from the result set into the specified one collection or more collections unless restricted by the `LIMIT` clause:
```

Comment:
```suggestion
You can use the `BULK COLLECT` clause with a `FETCH` statement. Instead of returning a single row at a time from the result set, the `FETCH BULK COLLECT` returns all rows at once from the result set into one or more specified collections unless restricted by the `LIMIT` clause:
```